src/EPR.Accreditation.Facade/Controllers/AccreditationController.cs

```diff
@@ -5,7 +5,7 @@
 namespace EPR.Accreditation.Facade.Controllers
 {
     [ApiController]
-    [Route("/api/[controller]/{accreditationExternalId}/")]
+    [Route("/api/[controller]/")]
```

Comment:
Why has this been removed? Other people have added this because they need it.

Reply:
Done.